modules/platforms/dotnet/Apache.Ignite.Core.Tests/Services/ServicesAsyncWrapper.cs

```diff
@@ -170,6 +170,7 @@ public ICollection<IServiceDescriptor> GetServiceDescriptors()
         }
 
         /** <inheritDoc /> */
+#pragma warning disable 618
```

Comment:
I'm not sure, but this `pragma` is the single change in the file, do we actually need it?

Reply:
Fixed